projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/Cisco_ignored.g4

```diff
@@ -237,6 +238,7 @@ null_block
       | TEMPLATE
       | TERMINAL
       | TIME_RANGE
+      | TIMEOUT
```

Comment:
This is a breaking change. It needs to be more granular. In particular, it interferes with `ts_timeout`.

Reply:
Sad.. it's true, I hadn't ran the unit tests yet. Alright, I'll fix it on Friday. Here are some of the line(s):

```
timeout xlate 3:00:00
timeout pat-xlate 0:00:30
... there are a bunch of other variations also
```

From ASA command reference: https://www.cisco.com/c/en/us/td/docs/security/asa/asa-command-reference/T-Z/cmdref4/t1.html?dtid=osscdc0002833#pgfId-1649892